src/transaction_builder.js

```diff
@@ -42,7 +42,7 @@ export const TimeoutInfinite = 0;
  * a payment to `destinationB`. The built transaction is then signed by `sourceKeypair`.</p>
  *
  * ```
- * var transaction = new TransactionBuilder(source)
```

Comment:
The second argument is supposed to be an object with a property called fee:

`new TransactionBuilder(source, { fee })`

Reply:
sorry I just realised that also - fixed